app/src/main/res/values/colors.xml

```diff
@@ -372,6 +372,7 @@
     <color name="mozac_feature_readerview_selected">@color/accent_high_contrast_dark_theme</color>
 
     <!-- Onboarding colors -->
+    <color name="onboarding_padded_background_color">#1F000000</color>
```

Comment:
Maybe we should use a more generic name as that same solution could be use on similar cases see https://github.com/mozilla-mobile/fenix/issues/17297

Reply:
I would propose that be a suggested follow up fix for #17297 since this is a PR from a contributor.

Reply:
👍🏽